utils/githooks/README.md

```diff
@@ -0,0 +1,19 @@
+# Git hooks
+
+This directory contains githooks that can be employed locally to
+automate things like copyright updates on modified files
+
+To use the pre-commit hook here, do the following locally
```

Comment:
(lint) Use semicolon or linefeed before 'do' (or quote to make it literal). [SC1010]